source/changelog.txt

```diff
@@ -181,6 +181,10 @@ New features in this release:
   ``// comments`` are now preserved as ``/* comments */`` in the
   history entry.
 
+- The :ref:`aggregation-pipeline` is now an optional parameter for
```

Comment:
[suggestion]

The way this currently renders makes it a bit tricky to parse. 

Spitballing:

"The :ref:`pipeline parameter <aggregation-pipeline>` paramater is now optional for :method:`db.collection.watch()`, :method:`db.watch()`, and :method:`Mongo.watch()`."

Reply:
Done